### ISSUE TITLE
NTR: Mucosal plaques (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8006]
New term request Mucosal plaques (segal_120412165814):   UMLS: 0455783

**Comment by MickeySegal** [comment]
Yes, for diseases such as White sponge nevus and Pemphigus vegetans

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001110
- label: Oral mucosa plaque
- parents: [Abnormal oral mucosa morphology (HP:0011830)](https://hpo.jax.org/app/browse/term/HP:0011830)
- pmid: 
- def: A raised, flat-topped lesion with a diameter of one centimeter or more that is located in the mucosa of the mouth.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8006
- synonyms:
